### DISCUSSION TITLE
How to get "weights" of WeightedEnsemble (TimeSeriesGreedyEnsemble)?

### DISCUSSION THREAD

**Discussion by jmren168** [autogluon/autogluon#3389] [Q&A] [answered]
Hi,

Is there any way go get the weights of a weighted ensemble predictor? Any clues are highly appreciated.

Best,

**Comment by shchur** [comment, marked as answer]
Hi, here is how this can be done in v0.8
```python
predictor = TimeSeriesPredictor(...).fit(...)
model_to_weight = predictor._trainer.load_model("WeightedEnsemble").model_to_weight
```

**Reply by jmren168** [reply]
Thanks for the quick reply. This also works in v0.7. Thanks again :)
